<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListaCobro</class>
 <widget class="QWidget" name="ListaCobro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista Cobro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#ffffff;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>-10</y>
     <width>651</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:55px; 
color:rgb(227, 227, 227);
letter-spacing: 0px;
font-weight: 700;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>COBROS DEL MES</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableListaCobro">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>471</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>640</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#downloadBtn{
background-color: #F5F5F5;
border-radius: 5px;
color: #392D33;
padding: 5px;
font-size: 20px;
letter-spacing: 2px;
text-align: center;
font-weight: 600;
}

#downloadBtn:hover{
	color: rgb(17, 17, 17);
	border:1px solid rgb(225, 225, 225);
}</string>
   </property>
   <property name="text">
    <string>Generar cobro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>640</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#exitBtn{
background-color: #F5F5F5;
border-radius: 5px;
color: #392D33;
padding: 5px;
font-size: 20px;
letter-spacing: 2px;
text-align: center;
font-weight: 600;
}

#exitBtn:hover{
	color: rgb(255, 0, 0);
	border:1px solid rgb(225, 225, 225);
}</string>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Total:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLista">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>550</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cobro_lista_message">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cobro_lista_message{
	color: rgb(47, 47, 47);
	font-size: 12px;
	text-align: center;
    font-weight: 500;
	letter-spacing: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
